Gopkg.toml

```diff
@@ -1,3 +1,7 @@
+required = [
+    "github.com/gogo/protobuf/proto",
```

Comment:
I don't think this should be necessary and I actually think adding it might result in extra files we don't want.

Reply:
`dep` still confuses me. it seems like you need this here once, to get `dep ensure` to succeed the first tine, after which maybe it can be removed.

Reply:
Removed.